components/cardList.js

```diff
@@ -34,12 +36,13 @@ export default function CardList({
     }
   };
 
-  useEffect(() => {
-    const unsubscribe = navigation.addListener("focus", () => {
+  useFocusEffect(
+    React.useCallback(() => {
```

Comment:
Nice i used the same thing to update post 👍🏻

Reply:
Nice!